README.md

```diff
@@ -832,30 +832,25 @@ cmd
 
 ### Unix
 
-Exiv2 can be built on many Unix and Linux distros.  We actively support the Unix Distributions NetBSD and FreeBSD
+Exiv2 can be built on many Unix and Linux distros.  We actively support the Unix Distributions NetBSD and FreeBSD.
 
-#### NetBSD
-
-NetBSD uses pkgsrc as the package manager.  You can build exiv2 with the commands:
+I am willing to support Exiv2 on commercial Unix distributions such as Solaris, AIX, HP-UX and OSF/1 provided you provide with an ssh account on your platform.  I will require super-user privileges to install software.
 
-```bash
-$ cd /usr/pkgsrc/graphics/exiv2
-$ make install
-```
+#### NetBSD
 
-You can build exiv2 from source using the methods described for linux.  I build and installed exiv2 using "Pure CMake" and didn't require conan.
-You will want to use `pkgsrc` to build/install
+You can build exiv2 from source using the methods described for linux.  I built and installed exiv2 using "Pure CMake" and didn't require conan.
+You will want to use the package manager `pkgsrc` to build/install:
 
 1) gcc  (currently GCC 5.5.0)<br>
```

Comment:
There is no need to add these `<br>` there. In the markdow language the item lists already create break lines automatically.

Reply:
I added this `<br>` to get it to format nicely on MarkDown application on the Mac.  If I don't use <br>, it puts all the bullets on a single line.  If I leave a blank line between the bullets, the document has ugly/useless blank lines.  Adding the `<br>` formatted the way I wanted it to look.

Reply:
That's odd (maybe a bug in the Mac application you are using?). If you click in the next link, you'll see how Github format the README in the web browser:

https://github.com/Exiv2/exiv2/tree/fix823_NetBSD_FreeBSD_support#5-6

In section 5.5 (Microsoft Visual C++) there are another item list without these `<br>`s and they are just shown fine, as the ones in the new section. 

I also opened the document with "Haroopad", the markdown editor I use, and I do not have such issue after removing them.

Reply:
![screenshot_19](https://user-images.githubusercontent.com/529982/57573477-ebc6e700-741f-11e9-9506-d04358979bf9.png)